Sources/SwiftDocC/Semantics/TechnologyBound.swift

```diff
@@ -9,7 +9,8 @@
 */
 
 /// An entity directly referring to the technology it belongs to.
+@available(*, deprecated, message: "This deprecated API will be removed after 6.2 is released")
```

Comment:
What protocol replaces this one? Has it been renamed or do we want to drop it completely?

Reply:
This protocol is never used anywhere. Nothing conforms to it and no argument or property references it.